tests/scripts/all.sh

```diff
@@ -171,6 +171,12 @@ pre_initialize_variables () {
     export MBEDTLS_TEST_OUTCOME_FILE
     export MBEDTLS_TEST_PLATFORM
 
+    if [[ "${JOB_TYPE}" != "PR" ]]; then
+        # Enable extended testing in the nightlies and release jobs. In future we might
+        # want to move this into the Groovy scripts.
+        export MBEDTLS_TEST_EXTENDED=1
```

Comment:
(minor, semantics) let's consider the nighties the "full" set of tests and the PR runs as "abridged", rather than the PR runs "full" and the nightlies "extended". i.e. can we call this `MBEDTLS_TEST_FULL`

Reply:
One last question, after re-reading this: is a positive or negative option best? i.e. do we want to require `MBEDTLS_TEST_FULL` to be set to get the full tests, or do we want a potential `MBEDTLS_TEST_ABRIDGED` to be set to get the shorter set of tests?

Wondering if it would be better to fail safe, i.e. if something goes wrong we get the longer tests rather than the shorter ones

Reply:
As it stands, the logic in this block is fail-safe: if `JOB_TYPE` is set to anything other than `PR`, or is unset, we will set `MBEDTLS_TEST_EXTENDED`.

We could certainly invert that to have `MBEDTLS_TEST_ABRIDGED`. No strong feelings on that.

Reply:
Happy with the way you've changed to always set `MBEDTLS_TEST_ABRIDGED`, and only skip tests if it has the value `1`. This feels "safe"